File: 58. Length of Last Word/Solution.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int size = s.size();
        int lastWordSize = 0;
        for (int i = size - 1; i >= 0; i--) {
            if (isspace(s[i]) && lastWordSize > 0) {
                break;
            }
            if (!isspace(s[i])) {
                lastWordSize++;
            }
        }
        return lastWordSize;
    }
};

File: 58. Length of Last Word/Solution.java
/**
 * Given a string s consisting of words and spaces, return the length of the last word in the string.
 * A word is a maximal substring consisting of non-space characters only.
 */

public class Solution {
    public int lengthOfLastWord(String s) {
        s = s.trim();
        return s.substring(s.lastIndexOf(" ") + 1).length();
    }
}